src/redux/sagas/cardsForDoctor.saga.ts

```diff
@@ -0,0 +1,44 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+import { AxiosResponse } from 'axios';
+import { createErrorNotificationMessage, utils } from './utils';
+import { cardsForDoctor, notificationError } from '../actions';
+import { AppointmentsForDoctor } from '../../resources/appointments/appointments.types';
+import appointments from '../../resources/appointments/appointments.api';
+import { ResolutionsResponse } from '../../resources/resolutions/resolutions.types';
+import resolutionsAPI from '../../resources/resolutions/resolutions.api';
+
+function* cardsForDoctorGet(action: ReturnType<typeof cardsForDoctor.pending>) {
+  try {
+    const { payload } = action;
+    const appointmentsResponse: AxiosResponse<AppointmentsForDoctor> = yield call(appointments.fetchAppointmentsForDoctor, payload.offset, payload.limit);
+    const resolutionResponse: AxiosResponse<ResolutionsResponse> = yield call(resolutionsAPI.fetchResolutions, payload.offset, payload.limit);
+    return {
+      cards: [...appointmentsResponse.data.appointments.map((appointment) => ({
```

Comment:
It could be moved to separate function to make return statement more readable.
Also, is it required to use [...data.map()] for this collection? Check Array.flat function if you want to drop one level of array.